CONTRIBUTING.md

```diff
@@ -277,19 +277,21 @@ package ...;
 #### Update Apache license header in modified files as necessary
 
 Always check the date range in the license header. For example, if you've
-modified a file in 2019 whose header still reads:
+modified a file in 2021 whose header still mentions Pivotal and/or an end year,
+like in the examples below:
```

Comment:
Do we need this as we have `2011-Present`?

Reply:
this was intended to accommodate for the possibility that in low-traffic projects we could still have headers without the `-Present`.